.github/component-label-map.yml

```diff
@@ -2,10 +2,18 @@ blog:
   - changed-files:
       - any-glob-to-any-file:
           - content/en/blog/**
-registry:
+docs:
   - changed-files:
       - any-glob-to-any-file:
-          - data/registry/**
+          - content/en/**
+docs:
+  - changed-files:
+      - any-glob-to-any-file:
+          - content/en/**
```

Comment:
Is there something I'm missing or do we have duplicate entries here?

Reply:
For consistency, should `docs` be triggered on all `content/**`
And then for consistency add `lang:XX` for any `content/XX/**`, even English?

Reply:
No duplicate entries, the diff is just looking strange since I also moved registry down.

The goal is to distinguish between translations and non-translations, such that I can have dedicated views with one label vs doing a "-lang:1 -lang:2 -lang:3..."

Reply:
If I look at the file source, I see two entries for `docs` that have the same fields, so I think that there is duplication. That is, in the following, lines 5 to 8 are the same as 9 to 12.

> <img width="403" alt="image" src="https://github.com/open-telemetry/opentelemetry.io/assets/4140793/a7937f98-c6c7-4905-bb44-32cf184bc105">

Anyhow, maybe we can drop the `docs` tagging rules and use `lang:en` for English pages? Would that work for the use cases you have in mind (I think that it does). That is, use `lang:en` and `-lang:en`. WDYT?

Reply:
oh... then I dropped the registry: somewhere, let me fix that, `lang:en` is a good idea, becaue there should (rarely) be a PR that touches multiple languages.